interfaces/builtin/udisks2.go

```diff
@@ -102,6 +102,7 @@ umount /{,run/}media/**,
 # give raw read access to the system disks and therefore the entire system.
 /dev/sd* r,
 /dev/mmcblk* r,
+/dev/vd* r,
```

Comment:
This is not related to the change (not that I dislike it but it confuses the premise of the PR)